### DISCUSSION TITLE
Ubuntu 22

### DISCUSSION THREAD

**Discussion by natzakaria** [datarhei/restreamer#608] [General]
Does restreamer work on Ubuntu 22?
Anyone successful installing it on a VPS?

**Comment by svenerbeck** [comment]
Hello natzakaria,

Yes, Restreamer can be installed on any system running Docker. Since Ubuntu 22 supports Docker, you should be able to install Restreamer without any issues, regardless of whether it's a VPS or any other type of machine.

For more information on the minimum requirements and how to install Restreamer, I'd happily refer you to our official documentation: minimum requirements.

I hope this helps, and good luck with your installation!

Sincerely,
datarhei.com

**Comment by natzakaria** [comment]
Thanks

I am running it on a VPS and works extremely well altho I was confused on
the ingest part of it.
It appears I would need to run nginx rtmp to create a rtmp url and input
that to restreamer.
I couldnt find a way for restreamer to generate its own rtmn url

But in any case its a very good program

Congratulations
